src/circllhist.c

```diff
@@ -705,7 +708,29 @@ hist_approx_count_below(const histogram_t *hist, double threshold) {
 }
 
 uint64_t
-hist_approx_count_above(const histogram_t *hist, double threshold) {
+hist_approx_count_below_exclusive(const histogram_t *hist, double threshold) {
+  int i;
+  uint64_t running_count = 0;
+  if(!hist) return 0;
+  ASSERT_GOOD_HIST(hist);
+  for(i=0; i<hist->used; i++) {
+    if(hist_bucket_isnan(hist->bvs[i].bucket)) continue;
+    double bucket_bound = hist_bucket_to_double(hist->bvs[i].bucket);
+    double bucket_upper;
+    if(bucket_bound < 0.0)
+      bucket_upper = bucket_bound;
+    else
+      bucket_upper = bucket_bound + hist_bucket_to_double_bin_width(hist->bvs[i].bucket);
```

Comment:
nit: This doesn't change the answer for `hist_approx_count_below_exclusive` but for `abs(bucket_bound) < 10`, the value returned by `hist_bucket_to_double_bin_width` is actually `width/10` as it doesn't account for lines 971 through 973 in `int_scale_to_hist_bucket`. Don't know if this is intentional.

Reply:
This sounds like a different bug, can you describe in more detail what you're seeing?  The internal encoding represents 1.0 to 9.9 as 10 to 99.  So the "width" [encoded 10, encoded 11) is 1 representing the 0.1 between [1.0, 1.1)

Reply:
I did see the inequality was off in below_exlcusive and the bucket_upper is equal to the lower bound of the greater bucket and we should be looking for things less than that, not less than or equal to.

Reply:
Your comment about internal encoding makes sense. Thank you for clearing that up.

Regarding the inequality, I believe what you had before was correct, unless I am misunderstanding something.